Modules/Core/Tests/Feature/ImportControllerTest.php

```
@@ -35,4 +37,368 @@ public function it_displays_import_page(): void
+
+        /** Act */
+        $this->actingAs($user);
+        $response = $this->get(route('import.delete', ['id' => 'invalid']));
+
+        /** Assert */
+        // Should handle gracefully
+        $this->assertTrue(
+            $response->isRedirect() || 
+            $response->getStatusCode() >= 400
+        );
+    }
```

PHPMD (2.15.0)

The method it_handles_delete_with_invalid_id_type is not named in camelCase. (undefined)

(CamelCaseMethodName)